sale_automatic_workflow/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2011 Akretion Sébastien BEAU <sebastien.beau@akretion.com>
+# Copyright 2013 Camptocamp SA (author: Guewen Baconnier)
+# Copyright 2016 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Automatic Workflow",
+    "version": "16.0.1.0.1",
+    "category": "Sales Management",
+    "license": "AGPL-3",
+    "author": "Akretion, "
+    "Camptocamp, "
+    "Sodexis, "
+    "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/sale-workflow",
+    "depends": ["sale_stock", "sales_team", "sale_delivery_state"],
```

Comment:
The sale_delivery_state dependency is no longer required.
Now in v16 we have the [delivery_status](https://github.com/OCA/OCB/blob/16.0/addons/sale_stock/models/sale_order.py#L34) field of the sale_stock module.